components/snackbar/snackbar.html

```diff
@@ -0,0 +1,110 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="../snackbar/snackbar.css" />
+    <title>Netra UI | Snackbar</title>
+  </head>
+  <body>
+    <header class="nav-container">
+      <div class="branding">
+        <a href="index.html"
+          ><img
+            class="brand_image"
+            src="https://cdn-0.emojis.wiki/emoji-pics/google/nazar-amulet-google.png"
+            alt="brand-image"
+        /></a>
+        <span class="brand_name">NETRA UI</span>
+      </div>
+      <nav class="navbar">
+        <ul class="nav-items">
+          <!-- <li><a href="">Home</a></li> -->
+          <li class="list-items item-1">
+            <a href="" class="doc">Documentation</a>
+          </li>
+          <li class="list-items item-2">
+            <a href=""><i class="fab fa-github github-icon"></i></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="wrapper">
+      <aside class="sidebar-container">
+        <div class="sidebar">
```

Comment:
I think this div is not required in the container there is only one element. Once check and remove